### PR TITLE
feat: idle members UI changes created

### PR DESCRIPTION
**Pull request by harshith-venkatesh**

### CODE REVIEW

**Comment by harshith-venkatesh** [comment]
https://www.loom.com/share/d4c18cabffb94fdcb4c586ef1f120865

**Comment by harshith-venkatesh** [comment]
image

File: src/components/idleMembers/card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import classNames from '@/components/idleMembers/card/card.module.scss';
+
+type Props = {
+  idleMemberUserName: string
+}
+
+const Card: FC<Props> = ({ idleMemberUserName }) => {
+  const imageGenerator = (name: string) => `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members/${name}/img.png`;
```
Comment (sumitd94) [inline src/components/idleMembers/card/index.tsx:9]: I think we have this URL defined in the .env file, why not take it from there

ref - https://github.com/Real-Dev-Squad/website-status/blob/develop/.env

File: src/components/idleMembers/section/index.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        </span>
+      )
+    }
+    {
+      (isLoading)
+        ? (
+          <span className={classNames.statusMessage}>Loading...</span>
+        )
+        : (
+          <div className={classNames.cardContainer}>{cards(content)}</div>
```
Comment (sumitd94) [inline src/components/idleMembers/section/index.tsx:36]: NIT: we can rename `cards` to `renderCards`
makes the purpose of the function clear.

File: src/components/idleMembers/section/section.module.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+.section {
+  height: 100%;
+  min-width: 17rem;
+  font-size: 1rem;
+}
+
+.cardContainer {
+  display: flex;
+  flex-direction: row;
```
Comment (sumitd94) [inline src/components/idleMembers/section/section.module.scss:17]: NIT: Not needed, by default `display: flex,` assumes the direction to be row.

**Review by sumitd94 — CHANGES_REQUESTED** [review]
Few NIT Comments, rest all looks good to me. Thanks.

**Review by sumitd94 — DISMISSED** [review]
Looks good to me, Thanks

**Review by whyDontI — CHANGES_REQUESTED** [review]
Nice work @harshith-venkatesh 😃 

A little change,
can you center the cards?

**Comment by harshith-venkatesh** [comment]
@whyDontI @sumitd94 can we merge this
It is still pending

**Review by whyDontI — APPROVED** [review]
Nice work @harshith-venkatesh 🥳

**Pull request merged**